data/human/hails.txt

```diff
@@ -2195,6 +2195,357 @@ phrase "friendly civilian"
 		`Apparently they want me to pay 5% of my paycheck for union dues! I'm gonna keep it. My boss is a good guy; he'll look out for me.`
 		`I saved up and bought extra bunk space so my wife could come on trips with me, and now she says "we need to talk." That can't be a good sign, can it?`
 
+phrase "friendly civilian"
+	word
+		"If you want to "
+		"To "
+	word
+		"make"
+		"turn"
+		"earn"
+		"get"
+	word
+		" "
+	word
+		"a profit"
+		"some credits"
+		"good money"
+		"a good turnover"
+		"a living"
+	word
+		" "
+	word
+		"while mining"
+		"as an asteroid miner"
+		"mining asteroids"
+		"in the asteroid mining business"
+	word
+		", "
+	phrase
+		"miner advice: outfit"
+		"miner advice: mineral"
+		"miner advice: ship"
+		"miner advice: other"
+	word
+		"." 3
+		"!"
+	replace
+		"turn some" "turn over some"
+		"turn good" "turn over good"
+		"turn a good turnover" "make a good turnover"
+		"%I" "i"
```

Comment:
Not seeing where this would be utilized. Was this leftover from a specific `word` that was removed?

Reply:
Yeah, these hails went through a lot of iterations. I'll remove it.